### PR TITLE
feature/WDP190703-34/new-furniture-RWD 

### PR DESCRIPTION
**Pull request by danielmark0116**
Implement RWD to the new-furniture menu.

Also created additional mixin for mobile devices (both tablets and phones)

### CODE REVIEW

File: src/sass/components/_panel-bar.scss [hunk 1]
```diff
@@ -27,8 +33,11 @@
     }
   }
 
+  position: relative;
```
Comment (matwjablonski) [inline src/sass/components/_panel-bar.scss]: this is duplicated in line 9

File: src/sass/components/_panel-bar.scss [hunk 2]
```diff
@@ -89,5 +124,23 @@
         }
       }
     }
+
+    &.new-furniture-dots {
+      @include mobile {
+        width: 100%;
+        display: block;
+        // margin-top: 15px;
```
Comment (matwjablonski) [inline src/sass/components/_panel-bar.scss]: is this necessary?

Comment (danielmark0116) [reply src/sass/components/_panel-bar.scss]: I left only width - without it the dots won't move to the new line, below tabs on mobiles

File: src/sass/components/_panel-bar.scss (hunk 1 above)
Comment (danielmark0116) [reply src/sass/components/_panel-bar.scss]: I got rid of it eventually, it was doing nothing

**Pull request merged**